gridded_etl_tools/utils/publish.py

```diff
@@ -150,34 +146,44 @@ def to_zarr(self, dataset: xr.Dataset, *args, **kwargs):
             self.info("Exiting without parsing since the dataset manager was instantiated as a dry run")
             self.info(f"Dataset final state pre-parse:\n{dataset}")
         else:
-            # Don't use update-in-progress metadata flag on IPLD or on a dataset that doesn't have existing data stored
-            if not isinstance(self.store, IPLD):
-                # Update metadata on disk with new values for update_in_progress and update_is_append_only, so that if
-                # a Zarr is opened during writing, there will be indicators that show the data is being edited.
-                self.info("Writing metadata before writing data to indicate write is in progress.")
-                if self.store.has_existing:
-                    update_attrs = {
-                        "update_in_progress": True,
-                        "update_is_append_only": dataset.get("update_is_append_only"),
-                        "initial_parse": False,
-                    }
+            # Determine Zarr format
+            if "zarr_format" in kwargs:
```

Comment:
I think we `ValueError`  in the case that zarr_format is in the kwargs. That's supposed to be determined by the manager, and I can't think of a case where we'd want that to be overwritten